src/extension.js

```diff
@@ -359,15 +360,24 @@ function init() {
     Clipboard.watchService();
 }
 
+var lockscreenInput = null;
```

Comment:
Maybe just move this one up, so it's on the line after `let serviceIndicator = null;`

Reply:
Cool! Done in [faad0fa](https://github.com/GSConnect/gnome-shell-extension-gsconnect/pull/1575/commits/faad0fa42e7a57df767499af39e3784d4912f08f).